config/config/sc-config.yaml

```diff
@@ -238,6 +238,7 @@ prometheus:
 
   # Todo remove dependencie on alertmanager from service cluster
   alertmanagerSpec:
+    replicas: 3
```

Comment:
Do we really need 3? I think 2 should be enough if there are no other limitations

Reply:
A weird thing I realized is that it only seems to work if there are 3 actually... There seems to be some kind of weird leader-follower thing going on so the 3rd work while the other 2 is occupied.

Reply:
I see, well then it's fine I guess